gradio/utils.py

```diff
@@ -1056,24 +1063,10 @@ def tex2svg(formula, *_args):
 
 
 def abspath(path: str | Path) -> Path:
-    """Returns absolute path of a str or Path path, but does not resolve symlinks."""
-    path = Path(path)
-
-    if path.is_absolute():
-        return path
+    return Path(os.path.abspath(str(path)))
```

Comment:
Ok so the reason we used the [convoluted symlink logic](https://github.com/gradio-app/gradio/pull/3037) is because of this issue: https://github.com/gradio-app/gradio/issues/3024

Basically, we don't want to resolve symlinks but we do want to make them absolute. This fix might work, let me check

Reply:
Ok so this should work, nice! I would just add a test case in test_utils.py to confirm that symlinks are made absolute but not resolved.